django/apps/blue_management/blue_mgnt/views/utils.py

```diff
@@ -0,0 +1,24 @@
+
+OFFENDING_CSV_CHARS = set('@+-=|%')
+
+
+def is_injection(s):
+    return len(s) > 0 and s[0] in OFFENDING_CSV_CHARS
+
+
+def escape(payload):
+    '''
+    Escapes CSV payloads to prevent Excel injection.
+    Works only on str, and bypasses any other type.
+    '''
+    if isinstance(payload, str) and is_injection(payload):
+        payload = payload.replace("|", "\|")
```

Comment:
This looks good, but I wonder if we want to instead of using "|" hardcoded, make sure we are escaping whatever we use as the separator?